backend/backendserver/src/main/java/com/github/klucsik/birdnoiseserver/backendserver/controller/DevicePlayParamController.java

```diff
@@ -47,4 +46,19 @@
         service.delete(id);
         return new ResponseEntity<>(HttpStatus.OK);
     }
+
+    @GetMapping("/setToDeployable")
+    public ResponseEntity<String> setToDeployable(@RequestParam Long deviceId) {
+        return new ResponseEntity<>(service.setToDeployable(deviceId), HttpStatus.OK);
+    }
+
+    @GetMapping("/setToDraft")
+    public ResponseEntity<String> setToDraft(@RequestParam Long deviceId) {
+        return new ResponseEntity<>(service.setToDraft(deviceId), HttpStatus.OK);
+    }
+
+    @GetMapping("/setToDeleted")
+    public ResponseEntity<String> setToDeleted(@RequestParam Long deviceId) {
+        return new ResponseEntity<>(service.setToDeleted(deviceId), HttpStatus.OK);
+    }
```

Comment:
How this can select which DPP should be changed? One device should have multiple DPP